python/pyspark/ml/deepspeed/tests/test_deepspeed_distributor.py

```diff
@@ -0,0 +1,176 @@
+#
```

Comment:
should add this file into https://github.com/apache/spark/blob/master/dev/sparktestsupport/modules.py so the test actually runs.